packages/browser-destinations/destinations/fullstory/src/identifyUserV2/index.ts

```diff
@@ -0,0 +1,95 @@
+import type { BrowserActionDefinition } from '@segment/browser-destination-runtime/types'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import type { FS } from '../types'
+import { segmentEventSource } from '..'
+
+// Change from unknown to the partner SDK types
+const action: BrowserActionDefinition<Settings, FS, Payload> = {
+  title: 'Identify User',
```

Comment:
In our cloud mode (server side) equivalents of these actions, we included the version number in the title e.g. "Identify User V2". Does it make sense to do the same here?

Reply:
Totally makes sense. This is probably the display value to the user.  ![themoreyouknow](https://emoji.slack-edge.com/T02FE3LFK/themoreyouknow/9952896b519f1361.gif)